tests/publiccloud/check_services.pm

```diff
@@ -37,7 +37,7 @@ sub run {
 
     # cloud-init
     # cloud-init is notavailable in Micro
-    if ((is_azure || is_ec2) && !is_sle_micro) {
+    if (cloudinit_enabled) {
         record_info('cloud-init', $instance->ssh_script_output('systemctl --no-pager --full status cloud-init*', proceed_on_failure => 1));
```

Comment:
There is also `sudo cloud-init status`

Reply:
I am now testing `cloud-init status` but I'd like to keep `cloudinit_enabled()` where we can controll where is `cloud-init` expected and where not.

Reply:
in such a case `cloudinit_enabled` would be confusing name , let's rename it into `cloudinit_check_enabled`

Reply:
We agreed to rename this to `is_cloudinit_supported`.

Please recheck.